src/components/Filter/Filter.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable react/button-has-type */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/prop-types */
+
+import React, { useReducer } from 'react';
+import useFilter from '../hooks/useFilter';
+
+const initialState = {
```

Comment:
let's move it to the hooks folder and name `useFilter`

Reply:
You mean to transfer filter.js file to useFilter as a custom hook in the hook folder, am I right?